src/stage_node.cpp

```diff
@@ -268,6 +389,10 @@ int StageNode::SubscribeModels()
     vehicle->init(use_topic_prefix, this->one_tf_tree_);
   }
 
+  for (std::shared_ptr<Object> object: this->objects_) {
```

Comment:
Please be consistent with the type specifiers. Either put here auto or change the other loop heads.